turbopack/crates/turbo-tasks-backend/src/backend/mod.rs

```diff
@@ -201,33 +201,43 @@ impl TurboTasksBackendInner {
         unsafe { ExecuteContext::new_with_tx(self, tx, turbo_tasks) }
     }
 
+    #[inline]
     fn suspending_requested(&self) -> bool {
         (self.in_progress_operations.load(Ordering::Relaxed) & SNAPSHOT_REQUESTED_BIT) != 0
     }
 
+    #[inline]
     fn operation_suspend_point(&self, suspend: impl FnOnce() -> AnyOperation) {
-        if self.suspending_requested() {
+        #[cold]
```

Comment:
I think the cold annotation is fine here (I don't think it's likely to hurt, and it's something llvm can't really predict on its own without PGO), but I think in practice `#[cold]` tends not to have much benefit because the CPU's branch predictor is already pretty good at identifying cold branches like this.